.github/workflows/auto-merge-staging.yml

```diff
@@ -15,12 +15,8 @@ jobs:
     steps:
       - name: Checkout 🛎
         uses: actions/checkout@v3
-
-      - name: Merge `main` branch
-        uses: everlytic/branch-merge@1.1.2
-        id: branch-merge
-        with:
-          github_token: ${{  secrets.GITHUB_TOKEN }}
-          source_ref: 'main'
-          target_branch: 'staging'
-          commit_message_template: '[Automated] Merged {source_ref} into target: {target_branch}'
+      - name: Sync `staging`
+        run: |
+          git config user.name github-actions
+          git config user.email github-actions@github.com
+          git push origin --force main:staging
```

Comment:
🙋🏻 Does this `main:staging` syntax do the same thing as `git reset origin/main` would do if `staging` was checked out?

Reply:
Hmmm I'm not exactly sure, but the syntax here is:
```
git push origin --force [local branch]:[remote branch]
```
([docs](https://git-scm.com/docs/git-push#Documentation/git-push.txt-ltrefspecgt82308203))